content/docs/command-reference/push.md

```diff
@@ -181,11 +181,11 @@ One could do a simple `dvc push` to share all the data, but what if you only
 want to upload part of the data?
 
 ```dvc
-$ dvc push --with-deps matrix-train.p.dvc
+$ dvc push --with-deps test-posts
```

Comment:
we need to rewrite the example above, the project structure for this command to make sense?

Reply:
also, why do change the stage name here, intentional?

Reply:
> rewrite the example above, the project structure for this command to make sense

Yes! It was also done in the complementary PR. Both are merged, please check in https://dvc.org/doc/command-reference/push#example-with-dependencies

> why do change the stage name here, intentional?

Intentional. I came up with these new names. But now that you mention it `matrix-train` would've been the obvious choice here 🤦

Reply:
Wait no, that't the name of the next stage. That's shy I came up with `test-posts`.